app/main.py

```diff
@@ -1,8 +1,44 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
```

Comment:
Although PEP does not declare the annotation style for functions with multiple arguments, I would recommend using Google Styleguide for this purpose for better readability. Reformat `__init__` input here and for the `Car` class according to [this recommendation](https://google.github.io/styleguide/pyguide.html#3192-line-breaking).